tests/test_consistency.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import pytest
```

Comment:
filenames of test should be like test_{file_its_testing}. for ex here you are testing functionality provided in graphite_encoder.py so the filename of this test file should be test_graphite_encoder.py.